elementary/monitor/data_monitoring/alerts/integrations/integrations.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+
+from elementary.config.config import Config
+from elementary.exceptions.exceptions import Error
+from elementary.monitor.data_monitoring.alerts.integrations.base_integration import (
+    BaseIntegration,
+)
+from elementary.monitor.data_monitoring.alerts.integrations.slack.slack import (
+    SlackIntegration,
+)
+from elementary.tracking.tracking_interface import Tracking
+
+
+class UnsupportedAlertIntegrationError(Error):
+    """Exception raised while executing edr monitor without a supported integration"""
+
+    def __init__(self) -> None:
+        super().__init__(
+            "Failed to run edr monitor - no supported integration was provided"
+        )
+
+
+class Integrations:
+    @staticmethod
+    def create_integration(
```

Comment:
nitpicking 🤓 but maybe it's cause I'm  dealing a lot with ORM right now but I feel it should be "get" and not "create"